muxers/mplex/src/lib.rs

```diff
@@ -174,6 +175,9 @@ where
 }
 
 /// Multiplexer. Implements the `StreamMuxer` trait.
+///
+/// This implementation isn't capable of detecting when the underlying socket changes its address,
+/// and no [`StreamMuxerEvent::AddressChange`] event is ever emitted.
```

Comment:
This may eventually change with [Multipath TCP](https://en.wikipedia.org/wiki/Multipath_TCP).